client/encointer-api-client-extension/src/ceremonies.rs

```diff
@@ -326,7 +326,7 @@ impl CeremoniesApi for Api {
 		// get stats of every meetup
 		for m in 1..=mcount {
 			let m_location = self.get_meetup_location(&community_ceremony, m)?.unwrap();
-			let time = self.get_meetup_time(m_location, ONE_DAY)?;
+			let time = self.get_meetup_time(m_location, ONE_DAY).unwrap_or(0);
```

Comment:
What is the motivation here?

Reply:
needed that to make sth work, but will remove as it seems not necessary now